Web identity tokens cannot be used without the sts service module

**Describe the Bug**

The error `To use web identity tokens, the 'sts' service module must be on the class path.` occurs when trying to use web identity tokens. 

**How to Reproduce**

Explicitly set the AWS credentials to be the web identity token file, then setup the environment to use it. It will fail

**Expected Behavior**

No error should occur and the service and successfully use web identity tokens as AWS credentials to use AWS resources

**Additional Context**

Error:

```
Caused by: java.lang.ClassNotFoundException: software.amazon.awssdk.services.sts.internal.StsWebIdentityCredentialsProviderFactory
	at loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
	at lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at base/java.lang.Class.forName0(Native Method) ~[?:?]
	at base/java.lang.Class.forName(Class.java:375) ~[?:?]
	at util.ClassLoaderHelper.loadClass(ClassLoaderHelper.java:114) ~[sdk-core-2.29.6.jar:?]
	at util.ClassLoaderHelper.loadClass(ClassLoaderHelper.java:73) ~[sdk-core-2.29.6.jar:?]
	at internal.WebIdentityCredentialsUtils.factory(WebIdentityCredentialsUtils.java:45) ~[auth-2.29.6.jar:?]
	at credentials.WebIdentityTokenFileCredentialsProvider.<init>(WebIdentityTokenFileCredentialsProvider.java:113) ~[auth-2.29.6.jar:?]
	at credentials.WebIdentityTokenFileCredentialsProvider.<init>(WebIdentityTokenFileCredentialsProvider.java:50) ~[auth-2.29.6.jar:?]
	at credentials.WebIdentityTokenFileCredentialsProvider$BuilderImpl.build(WebIdentityTokenFileCredentialsProvider.java:313) ~[auth-2.29.6.jar:?]
	at credentials.WebIdentityTokenFileCredentialsProvider.create(WebIdentityTokenFileCredentialsProvider.java:133) ~[auth-2.29.6.jar:?]
	at aws.util.AwsConfig.loadCredentialsProvider(AwsConfig.java:86) ~[atleon-aws-util-0.31.2.jar:?]
	at aws.sqs.SqsConfig.buildClient(SqsConfig.java:37) ~[atleon-aws-sqs-0.31.2.jar:?]
	at sqs.SqsReceiverOptions.createClient(SqsReceiverOptions.java:99) ~[atleon-aws-sqs-0.31.2.jar:?]
	at sqs.SqsReceiver$Poller.<init>(SqsReceiver.java:115) ~[atleon-aws-sqs-0.31.2.jar:?]
	at aws.sqs.SqsReceiver.lambda$receiveManual$0(SqsReceiver.java:76) ~[atleon-aws-sqs-0.31.2.jar:?]
```

Add `aws-java-sdk-sts` to the class path should fix this I believe

--- a/aws/aws-util/pom.xml
+++ b/aws/aws-util/pom.xml
@@ -26,5 +26,12 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>regions</artifactId>
         </dependency>
+
+        <!-- Third Party Runtime Dependencies -->
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>sts</artifactId>
+            <scope>runtime</scope>
+        </dependency>
     </dependencies>
 </project>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -402,6 +402,11 @@
             </dependency>
             <dependency>
                 <groupId>software.amazon.awssdk</groupId>
+                <artifactId>sts</artifactId>
+                <version>${amazon-sdk.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
                 <artifactId>utils</artifactId>
                 <version>${amazon-sdk.version}</version>
             </dependency>

